mobile/src/navigation/app/app.navigation.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { FC } from 'react';
+import {
+  createDrawerNavigator,
+  DrawerNavigationOptions,
+  DrawerContentScrollView,
+  DrawerItemList,
+} from '@react-navigation/drawer';
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  ScrollView,
+} from '~/components/common/common';
+import { RootScreenName, AppColor } from '~/common/enums/enums';
+import { ParamListBase } from '~/common/types/types';
+import { styles } from './styles';
+import {
+  DrawerNavigationHelpers,
+  DrawerDescriptorMap,
+} from '@react-navigation/drawer/lib/typescript/src/types';
+import { DrawerNavigationState } from '@react-navigation/native';
+
+// TODO: replace example components
```

Comment:
Can you declare these components in arrow fns style, and even if its just a mocks for real components - it would be better to split them to their own directories, I think